os/fs/smartfs/smartfs_utils.c

```diff
@@ -1193,13 +1202,13 @@ int smartfs_finddirentry(struct smartfs_mountpt_s *fs, struct smartfs_entry_s *d
 					/* Test if the name matches */
 
 					if (strncmp(entry->name, fs->fs_workbuffer, fs->fs_llformat.namesize) == 0) {
-						/* We found it!  If this is the last segment entry,
-						 * then report the entry.  If it isn't the last
-						 * entry, then validate it is a directory entry and
+						/* We found it!  If this is the last segment entry, then report the entry
+						 * If this segment is followed only by and extra '/' character in the end,
```

Comment:
I guess it will be  "only by an extra" rather than " only by and extra " .

Reply:
Rectified, thanks